Lecture2/Task4/com/playtika/ArrayAndLinkedLists.java

```diff
@@ -0,0 +1,122 @@
+package com.playtika;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Random;
+
+public class ArrayAndLinkedLists {
+    public static void main(String[] args) {
+        int a = 0;
+        final Random newRandom = new Random(); // Create Random generator
+
+        List<Integer> createdArrayList = new ArrayList<>();
+        List<Integer> createdLinkedList = new LinkedList<>();
```

Comment:
Please double check the task description.... It seems like your list isn't initialized properly